src/include/duckdb.h

```diff
@@ -14,6 +14,14 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+#ifdef _WIN32
+#define DUCKDBAPI __declspec(dllexport)
+#define DUCKDBCALL __cdecl
```

Comment:
One question: What does this `#define` achieve? It seems unused below.

Reply:
DUCKDBCALL is for exporting function pointers if they get added to the C API in the future